lib/IDE/TypeCheckCompletionCallback.cpp

```diff
@@ -47,6 +47,10 @@ void TypeCheckCompletionCallback::fallbackTypeCheck(DeclContext *DC) {
 
 Type swift::ide::getTypeForCompletion(const constraints::Solution &S,
                                       ASTNode Node) {
+  if (auto ContextualType = S.getContextualType(Node)) {
+    return ContextualType;
+  }
+
```

Comment:
Nit: There's another check for `getContextualType` a bit further down, seems like that can be removed now :)

Reply:
Good catch. Thanks